recipes/client.rb

```diff
@@ -1,19 +1,2 @@
 # frozen_string_literal: true
-#
-# Cookbook:: postgresql
-# Recipe:: client
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-postgresql_client_install 'postgresql client'
+Chef::Log.warn('The postgresql::client recipe has been deprecated and will be removed in the next major release of the cookbook. Please use the postgresql_server_install resource instead')
```

Comment:
if it's just deprecated you should probably keep the client install behavior. Otherwise you might as well just remove the whole recipe now

Reply:
Yeah I'm happy to remove all the recipes now tbh.